### PR TITLE
39 Add step id to schema

### PR DESCRIPTION
**Pull request by frederikja163**

### CODE REVIEW

**Review by rmadsen-ks — CHANGES_REQUESTED** [review]
Could we add to the unit test, that we verify that the step id is actually added? Maybe we could also test that the step id actually corresponds to the right step.

File: Parquet.Tests/ParquetFileTests.cs [hunk 1]
```diff
@@ -120,12 +121,14 @@ public void MergingFilesTest()
...
                 Assert.That(columns2.First().Data.GetValue(0), Is.EqualTo("ResultName2"));
-                Assert.That(columns1.Skip(1).First().Data.GetValue(0)?.ToString(), Is.EqualTo(stepGuid1.ToString()));
-                Assert.That(columns2.Skip(1).First().Data.GetValue(0)?.ToString(), Is.EqualTo(stepGuid2.ToString()));
+                Assert.That(columns1.Skip(1).First().Data.GetValue(0)?.ToString(), Is.EqualTo(runGuid1.ToString()));
+                Assert.That(columns2.Skip(1).First().Data.GetValue(0)?.ToString(), Is.EqualTo(runGuid2.ToString()));
                 Assert.That(columns1.Skip(2).First().Data.GetValue(0)?.ToString(), Is.EqualTo(parentGuid1.ToString()));
                 Assert.That(columns2.Skip(2).First().Data.GetValue(0)?.ToString(), Is.EqualTo(parentGuid2.ToString()));
                 Assert.That(columns1.Skip(3).First().Data.GetValue(0)?.ToString(), Is.EqualTo(null));
-                Assert.That(columns2.Skip(3).First().Data.GetValue(0)?.ToString(), Is.EqualTo("Test"));
+                Assert.That(columns2.Skip(3).First().Data.GetValue(0)?.ToString(), Is.EqualTo(stepGuid.ToString()));
```
Comment (frederikja163) [inline Parquet.Tests/ParquetFileTests.cs:129]: We have a test here that the step is actually added. But we can try doing it in a whole plan run too, if that is what you are talking about.

**Comment by frederikja163** [comment]
The tests fail right now, but once #47 has been merged it should fix the tests

**Review by rmadsen-ks — APPROVED** [review]
Looks good!

**Pull request merged**